### ISSUE TITLE
TopicLine shows Twitter icon for New Posts

### ISSUE THREAD

**Issue by Zero2Cool** [YAFNET/YAFNET#484]
I needed more control with the TopicLine so I used the existing control. However, after making the changes I noticed instead of saying New Posts, it displays a Twitter icon.

![image](https://user-images.githubusercontent.com/5612416/69488811-13b5ef00-0e34-11ea-8308-94b9ad1a8d2b.png)

Looks like this is the offending code in the TopicLine.cs code behind. But, for some reason, I can't get the solution to build so I can't even make a quick change lol
![image](https://user-images.githubusercontent.com/5612416/69488843-b3737d00-0e34-11ea-9f9d-fbf732b05edd.png)

In the topics control (in pages) I replaced this
``<%# this.CreateTopicLine((System.Data.DataRowView)Container.DataItem) %>``

with 

``<YAF:TopicLine runat="server" AltLastPost="<%# this.LastPostImageTT %>" DataRow="<%# Container.DataItem %>" />``

**Comment by w8tcha** [comment]
Dont use the topicLine.ascx this is only used for moderating pages, and its obsolete

**Comment by Zero2Cool** [comment]
Unfortunately the layout forces me to. The way it's designed takes up too
much space and isn't flexible for change.

**Comment by w8tcha** [comment]
Be aware that the duplicate control will be removed at one point. It really causes some problems. And its not working on the mytopics page.

**Comment by Zero2Cool** [comment]
Thanks for the heads up. Is there going to be a new control that the UI can be modified without having to do a solution rebuild?

**Comment by w8tcha** [comment]
No sorry this was the only solution to fix the postback issues. But can't you make the layout changes via css?

**Comment by Zero2Cool** [comment]
I don't think so. I had to tweak a lot of things to condense the layout. Much of it is reliant on Bootstrap and if I change it in the theme, then it changes it everywhere.  That also makes distinguishing the Quoted comments difficult too. 

I would like less space used up so less scrolling is needed to see more of the forum content. So, for the Topics, this is what I ended up with by altering the UI in the TopicLine control.
![image](https://user-images.githubusercontent.com/5612416/69495602-62907280-0e8e-11ea-8bbf-74ea44c52e83.png)


![image](https://user-images.githubusercontent.com/5612416/69495638-c1ee8280-0e8e-11ea-89b7-49e261c8bd81.png)
